lib/ES/Repo.pm

```diff
@@ -60,7 +70,7 @@ sub update_from_remote {
         unless ( $self->_try_to_fetch ) {
```

Comment:
Right now if we don't check that the reference is the same as the last execution in `_try_to_fetch`. We probably should update it if it has changed. That isn't super likely, but it is probably worth doing.